report/2.Analysis/EmbeddedPlatforms.tex

```diff
@@ -0,0 +1,78 @@
+% !TeX root = ../main.tex
+% This section is meant to introduce single-board computers that are often used for introducing programmers to beginners.
+% The section itself might not be used a whole lot during the design and implementation, as these are not the arduino platform.
+% Still, it is good to give examples of implementations.
+\section{Embedded platform}
+In the following section will examine the embedded platforms, that will be used in the project.
+For the 5th semester of the Software education, a Lego Mindstorms kit is supplied to each project group.
+Therefore the analysis will primarily be based around the Lego Mindstorms platform, with a few additional platforms also considered.
+
+To determine the best platform, the requirements for the project will need to be considered.
+\begin{enumerate}
+	\item At the very least, the platform must be able to perform a targeting calculation, based on a trained machine learning model.
+	\item Be able to interface with motors, most likely Lego.
+	\item Potentially be able to train a machine learning model, which is used for the prediction on which the targeting is based.
+\end{enumerate}
+
+As image recognition is an expensive computation, the current idea is to offload this to an external server or computer, which will compute the current object position and send that to the embedded device.
```

Comment:
se forrige kommentar. Jeg bryder mig ikke om at sige "the current idea is" hvis vi ikke forklarer hvorfor etc. hvilket i så fald skal stå forinden denne del.